### DISCUSSION TITLE
Is there a way to see IO per process?

### DISCUSSION THREAD

**Discussion by fdw** [aristocratos/btop#822] [Q&A]
Similar to `iotop`, I would like to see how much a process reads or writes, and find the top consumer. Is that already possible?

**Comment by foux** [comment]
> Similar to `iotop`, I would like to see how much a process reads or writes, and find the top consumer. Is that already possible?

Did you ever found a way @fdw  ?

**Reply by fdw** [reply]
Nope, unfortunately not.
